packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -0,0 +1,1114 @@
+		// We prefer to manually allow console logs despite the app being a CLI to catch wild/unwanted console logs better
+		// eslint-disable-next-line no-console
+		console.log(
+			`\n${chalk.bgGray(` ${chalk.bold.white("Slice Machine")} `)} ${chalk.dim(
+				"→",
+			)} Init command started\n`,
+		);
+
+		await this.manager.telemetry.initTelemetry();
+		await this.manager.telemetry.track({
+			event: "command:init:start",
+			repository: this.options.repository,
+		});
+
+		try {
+			await this.detectEnvironment();
+
+			assertExists(
+				this.context.framework,
+				"Project framework must be available through context to proceed",
+			);
+
+			await this.beginCoreDependenciesInstallation();
+			await this.loginAndFetchUserData();
+
+			if (this.options.repository) {
+				await this.useRepositoryFlag();
+			} else {
+				await this.selectRepository();
+			}
+
+			assertExists(
+				this.context.repository,
+				"Repository selection must be available through context to proceed",
+			);
+
+			if (!this.context.repository.exists) {
+				await this.createNewRepository();
+			}
+
+			await this.finishCoreDependenciesInstallation();
+			await this.upsertSliceMachineConfigurationAndStartPluginRunner();
+			await this.pushDataToPrismic();
+			await this.initializeProject();
+			await this.initializePlugins();
+		} catch (error) {
+			await this.trackError(error);
+
+			throw error;
+		}
+
+		await this.manager.telemetry.track({
+			event: "command:init:end",
+			framework: this.context.framework.prismicName,
+			repository: this.context.repository.domain,
+			success: true,
+		});
+
+		// We prefer to manually allow console logs despite the app being a CLI to catch wild/unwanted console logs better
+		// eslint-disable-next-line no-console
+		console.log(
+			`\n${chalk.bgGreen(` ${chalk.bold.white("Slice Machine")} `)} ${chalk.dim(
+				"→",
+			)} Init command successful!`,
+		);
+
+		try {
+			// We prefer to manually allow console logs despite the app being a CLI to catch wild/unwanted console logs better
+			// eslint-disable-next-line no-console
+			console.log(`
+  YOUR REPOSITORY
+    Dashboard            ${chalk.cyan(
+			`https://${this.context.repository.domain}.prismic.io`,
+		)}
+    API                  ${chalk.cyan(
+			`https://${this.context.repository.domain}.cdn.prismic.io/api/v2`,
+		)}
+
+  RESOURCES
+    Documentation        ${chalk.cyan(
+			this.context.framework.prismicDocumentation,
+		)}
+    Getting help         ${chalk.cyan("https://community.prismic.io")}
+
+  GETTING STARTED
+    Start Slice Machine  ${chalk.cyan(
+			this.context.projectInitialization?.patchedScript
+				? await getRunScriptCommand({
+						agent: this.context.packageManager || "npm",
+						script: "slicemachine",
+				  })
+				: await getExecuteCommand({
+						agent: this.context.packageManager || "npm",
+						script: "start-slicemachine",
+				  }),
+		)}`);
+		} catch {
+			// Noop, it's only the final convenience messsage
```

Comment:
tackError here ? then it could be removed from further up?

Reply:
Could you explain what you mean? Do you mean we should add `trackError()` to this `catch` block?

Reply:
This try catch block : ) 
https://github.com/prismicio/slice-machine/pull/799/files#diff-9bd9b7346b4e79315a4216e41c958e36d13508345728e46c470ec083271d3be5R134

Could they be combined into one try/catch block?